src/NServiceBus.Hosting.Windows/StartableAndStoppable/StartableAndStoppableTask.cs

```diff
@@ -0,0 +1,27 @@
+namespace NServiceBus
+{
+    using System.Threading.Tasks;
+    using Features;
+    using ObjectBuilder;
+
+    class StartableAndStoppableTask : FeatureStartupTask
+    {
+        StartableAndStoppableRunner startAndStoppablesRunner;
+
+        public StartableAndStoppableTask(IConfigureComponents configurer, IBuilder builder, IHoldStartAndStoppables startAndStoppablesBag)
```

Comment:
Something with the dependencies is really off in this design. Now this class requires the configurer and the builder. I'll see if I can send in a proposal PR or work with @colin-higgins directly